test/upgrade/test-upgrade.bats

```diff
@@ -105,15 +108,18 @@ podman \$opts run -d --name myrunningcontainer --label mylabel=$LABEL_RUNNING \
 
 podman \$opts pod create --name mypod
 
-podman \$opts network create mynetwork
+podman \$opts network create --disable-dns mynetwork
 
 echo READY
 while :;do
     if [ -e /stop ]; then
         echo STOPPING
         podman \$opts stop -t 0 myrunningcontainer || true
         podman \$opts rm -f     myrunningcontainer || true
-        podman \$opts network rm -f mynetwork
+        # sigh, network rm fails with exec: "ip": executable file not found in $PATH
+        # we cannot change the images afterwards so we remove it manually (#11403)
+        # hardcode /etc/cni/net.d dir for now
+        podman \$opts network rm -f mynetwork || rm /etc/cni/net.d/mynetwork.conflist
```

Comment:
Suggestion: `rm -f` will exit 0 even if the file doesn't exist. (I don't know under what circumstances that could happen, but it doesn't seem like an error worth failing the test for)